api/src/main/java/org/cardanofoundation/rosetta/api/construction/service/CardanoConstructionServiceImpl.java

```diff
@@ -203,23 +203,16 @@ public Integer calculateTxSize(NetworkIdentifierType networkIdentifierType,
     UnsignedTransaction unsignedTransaction;
     try {
       unsignedTransaction = createUnsignedTransaction(networkIdentifierType,
-          operations, ttl, !ObjectUtils.isEmpty(depositParameters) ? depositParameters
-              : new DepositParameters(Constants.DEFAULT_KEY_DEPOSIT.toString(),
+          operations, ttl, depositParameters != null ?
```

Comment:
DepositParameters could be an empty object

Reply:
Reverted